packages/contracts/src/RelayHub.sol

```diff
@@ -381,6 +386,15 @@ contract RelayHub is IRelayHub, Ownable {
         }
     }
 
+    function calculateDevCharge(uint256 charge) public view returns (uint256){
+        if (config.devFee == 0){ // save some gas in case of zero dev charge
+            return 0;
+        }
+        unchecked {
+        return charge.mul(config.devFee).div(100);
```

Comment:
`unchecked` only works on your current function, not inner functions.
it will only take effect if you do `charge*config.devFee/100`

Reply:
Fixed